x-pack/plugin/kql/src/main/java/org/elasticsearch/xpack/kql/parser/KqlParser.java

```diff
@@ -25,22 +24,17 @@
 public class KqlParser {
     private static final Logger log = LogManager.getLogger(KqlParser.class);
 
-    public QueryBuilder parseKqlQuery(String kqlQuery, SearchExecutionContext searchExecutionContext) {
-        if (log.isDebugEnabled()) {
+    public QueryBuilder parseKqlQuery(String kqlQuery, KqlParsingContext kqlParserContext) {
+        if (log.isTraceEnabled()) {
             log.debug("Parsing KQL query: {}", kqlQuery);
         }
```

Comment:
These logging levels don't make sense. Either its a trace or its debug. Not both.

Reply:
Fixed (log level is now trace and the conditional has been removed).